pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -232,8 +232,16 @@ func TestReconcileProvisionOK(t *testing.T) {
 
 		// test
 		reconcile(r, req)
+
 		checkReadyCond(t, fakeClient, username, corev1.ConditionTrue, "Provisioned")
 	})
+
+	t.Run("nstmplset_not_found", func(t *testing.T) {
+		r, req, _ := prepareReconcile(t, username)
+
+		// test
+		reconcile(r, req)
```

Comment:
We need to check the status error here, right?

Reply:
This is not_found test, so there is no NSTmplSet object available against which we can check the status.

Reply:
Ahh.. I see. I checked this code from my phone so couldn't see what `reconcile(r, req)` actually does. But now I see that it already does all needed assertions. looks good!